lib/chorde/decorators.py

```diff
@@ -78,12 +79,12 @@ def _make_namespace(f, salt = None, salt2 = None):
             body_digest.update(salt2)
         if fcode:
             body_digest.update(getattr(fcode, 'co_code', ''))
-        return "%s.%s#%s" % (mname,fname,body_digest.digest().encode("base64").strip("=\n"))
+        return "%s.%s#%s" % (mname,fname,b64encode(body_digest.digest()).strip("=\n"))
```

Comment:
Is the `strip` necessary here?

Reply:
Yes, if we want to generate the same namespaces. Could be an rstrip though.